targets/roslyn.props

```diff
@@ -6,7 +6,7 @@
     <NuGetPackageVersion>2.3.2-beta1-61921-05</NuGetPackageVersion>
     <BuildNumber>20170721.5</BuildNumber>
     <CommitHash>ad0efbb6975c74e3e19051896be7ffcbb70d2122</CommitHash>
-    <BuildCommand>$(DotNetCommand) build $(ProjectDirectory)/CLI.sln /p:Configuration=$(Configuration) /p:OfficialBuild=true /p:BuildNumber=$(BuildNumber)</BuildCommand>
+    <BuildCommand>$(DotNetCommand) build $(ProjectDirectory)/CLI.sln /p:Configuration=$(Configuration) /p:OfficialBuild=true /p:BuildNumber=$(BuildNumber) /p:DotNetBuildFromSource='$(DotNetBuildFromSource)'</BuildCommand>
```

Comment:
Is `CLI.sln` the best name for this file?  It's your repo, so you can do what you want, but that doesn't seem like an obvious name to me.

Reply:
Long term definitely not. In the long term the Roslyn repo is going to have two solutions: roslyn.sln and compilers.sln. The latter will be used in the source-build repo. 

In the short term though we are under some heavy constraints. Mostly because we actively service out of our 2.0 branch due to VS. Hence we need VS escrow approval to change anything. This is why in our 2.0 branch I am leaving it as CLI.sln. It is completely new and I dont need directory approval to add.